en/docs/administer/using-symmetric-encryption.md

```diff
@@ -1,15 +1,59 @@
-# Using Symmetric Encryption
+# Configurations Related to Symmetric Key Encryption
 
-With symmetric encryption a single key will be shared for encryption and decryption of information. Even though, WSO2 Identity Server uses [asymmetric encryption](../../administer/using-asymmetric-encryption) by default, you can switch to symmetric encryption. 
```

Comment:
since symmetric encryption is now the default, should we include steps on how to switch back to asymmetric if required?

Reply:
Inorder to revert back to assymetric encryption, following need to be added to deployment.toml file.

[keystore]
userstore_password_encryption = "InternalKeyStore"

The above segment should be added before [keystore.primary] entry in the deployment.toml file.

Following can be added at the end of toml file
[system.parameter]
"org.wso2.CipherTransformation"="RSA/ECB/OAEPwithSHA1andMGF1Padding"

[encryption]
internal_crypto_provider = "org.wso2.carbon.crypto.provider.KeyStoreBasedInternalCryptoProvider"

@gomathyk lets update with those instructions as well.

Reply:
Updated doc with content.